addons/mail/models/mail_message.py

```diff
@@ -868,17 +859,19 @@ def _message_reaction(self, content, action):
             self.env["mail.message.reaction"].create(create_values)
         if action == "remove" and reaction:
             reaction.unlink()
-        self._bus_send_reaction_group(content)
+        self._bus_send_reaction_group(content, store)
 
-    def _bus_send_reaction_group(self, content):
+    def _bus_send_reaction_group(self, content, store: Store):
```

Comment:
That's not a correct pattern, `_bus_send_reaction_group` should do nothing but to send the correct data.

If we need a "store filler" in 2 places, then you should split this method into one that fills the store which is called in both places.